### PR TITLE
CreateHabit page functionality

### PR DESCRIPTION
**Pull request by jujuzitroni**
UPDATE: added css to the CreateHabit and HabitList page
https://plants-need-createhabit-kmdhha.herokuapp.com/create-habit

if you test this:
please excuse the horrendous layout, css is still on my to do list.
click the plus button to go to the form.
these is no ux feedback if you submit a habit yet. 
please click the back button and see that your entry is added.
(also, i've not removed the console.log in the CreateHabit.tsx, so you can verify that the habit is complete with the data that will not appear in the client)

https://plants-need-createhabit-kmdhha.herokuapp.com/

part of #24 and #27

### CODE REVIEW

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

File: src/app/pages/CreateHabit/CreateHabit.stories.tsx [hunk 1]
```diff
@@ -6,4 +6,6 @@ export default {
   page: CreateHabit,
 };
```
Comment (MaxMuecket) [inline src/app/pages/CreateHabit/CreateHabit.stories.tsx:7]: parameters: {
    layout: 'fullscreen',
  },

**Review by MaxMuecket — APPROVED** [review]
Great work! ✌️

**Review by l-goede — APPROVED** [review]
Wow amazing:)

**Pull request merged**